hosting_docs/source/installation/3-apply-fsck-onreboot.rst

```diff
@@ -0,0 +1,83 @@
+.. _cchq-manual-install:
```

Comment:
This indicates that this is a new hyperlink, it's already used for actual install docs, so this should be removed.